### PR TITLE
Implement /wd/hub/session/:sessionId/element/:id/rect endpoint for better compatibility with Selenium 3

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Selenium 3 introduced new mandatory element property called _rect_. This PR adds the properly to Selendroid

### CODE REVIEW

File: selendroid-server/src/main/java/io/selendroid/server/android/internal/Rectangle.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.selendroid.server.android.internal;
+
+/**
+ * A copy of java.awt.Rectangle, to remove dependency on awt.
+ */
+public class Rectangle {
```
Comment (marcelogomez) [inline selendroid-server/src/main/java/io/selendroid/server/android/internal/Rectangle.java]: I don't think we really need this class Android already has a Rect implementation
https://developer.android.com/reference/android/graphics/Rect.html

File: selendroid-server/src/main/java/io/selendroid/server/model/AndroidNativeElement.java [hunk 2]
```diff
@@ -478,6 +479,12 @@ public Point getLocation() {
     return new Point(xy[0], xy[1]);
   }
 
+  @Override
+  public Rectangle getRect() {
+    Point location = getLocation();
```
Comment (marcelogomez) [inline selendroid-server/src/main/java/io/selendroid/server/model/AndroidNativeElement.java]: Can you take a look at the `click` implementation? Turns out that `getLocation` might not always work immediately :/

I think we can do some refactoring to make it safer

**Comment by marcelogomez** [comment]
Thanks for taking a stab at this :)
I added a couple of comments

**Comment by mykola-mokhnach** [comment]
@marcelogomez This PR was created more than half a year ago. It looks like the project is not maintained for quite a while. Do you plan to support it in the future? Are there better alternatives?

**Pull request closed without merging**